ceph/ceph_admin/helper.py

```diff
@@ -831,6 +832,57 @@ def validate_log_file_after_enable(cls):
     return True
 
 
+def validate_log_rotate(cls):
+    """
+    Verify logrotation of cephadm.log file
+    Args:
+        cls: cephadm instance object
+
+    Returns:
+        boolean
+    """
+    # step 1: check for the files in current log directory
+    out = cls.installer.exec_command(cmd="ls /var/log/ceph", sudo=True)
```

Comment:
One suggestion, can we directly check whether "/var/log/ceph/cephadm.log" is present using '-f' or similar rather than parsing the ls output. Please ignore, if I got it wrong

Reply:
+1, in this case `exec_command` will raise exception and validation for files is not needed. Also, I would suggest to use `out, _` to collect `exec_command` return values, as it returns tuple of stdout and err values

Reply:
Done.